flash/text/question_answering/collate.py

```diff
@@ -96,7 +96,7 @@ def _prepare_val_features(self, samples: Any, tokenized_samples: Any, pad_on_rig
         sample_mapping = tokenized_samples.pop("overflow_to_sample_mapping")
 
         # For evaluation, we will need to convert our predictions to substrings of the context, so we keep the
-        # corresponding example_id and we will store the offset mappings.
+        # corresponding example_id, and we will store the offset mappings.
```

Comment:
I think in this PR, we made a change like: `parameters' dictionary'. If that's right, probably it should be `offset's mappings` here (if we mead `mappings of the offset` here and not `offset mappings` term)? Not a native English speaker, so not too sure.

Reply:
mind send suggestion :)